vmdb/app/models/miq_queue.rb

```diff
@@ -441,6 +441,7 @@ def m_callback(msg, result)
 
   def requeue(options = {})
     options.reverse_merge!(self.attributes.symbolize_keys)
+    options.delete :id
```

Comment:
And so begins seattle.rb style... :trollface: 

:+1: This is a bug... @tenderlove what would it actually do if you tried to re-use an existing id?  Both on 3.2 and 4.2?